linera-execution/src/graphql.rs

```diff
@@ -16,7 +16,10 @@ use linera_base::{
 use linera_views::{common::Context, views::ViewError};
 use std::collections::BTreeMap;
 
-doc_scalar!(ApplicationId, "A unique identifier for an application");
+doc_scalar!(
+    GenericApplicationId,
+    "A unique identifier for an application"
```

Comment:
We could clarify "for a user or the system application" here, to distinguish it from the other type.